source/representation.lisp

```diff
@@ -447,9 +465,7 @@
                     (map nil #'resolve thing))
                    (t
                     (loop for slot in (subclause-slots thing)
-                      do (resolve (slot-value thing slot)))
+			  do (resolve (slot-value thing slot)))
```

Comment:
@xrme Which indentation style should be used here?

Reply:
Since this is a whitespace-only change, I would prefer not to see it.  If you changed something in the loop form, I wouldn't care which indentation style you decided to use.